crates/diesel_models/src/query/relay.rs

```diff
@@ -46,4 +46,18 @@ impl Relay {
         )
         .await
     }
+
+    pub async fn find_by_connector_reference_id(
+        conn: &PgPooledConn,
+        connector_reference_id: &str,
+        profile_id: &common_utils::id_type::ProfileId,
+    ) -> StorageResult<Self> {
```

Comment:
Can you rename this accordingly:

Either:
```suggestion
    pub async fn find_by_profile_id_connector_reference_id(
        conn: &PgPooledConn,
        profile_id: &common_utils::id_type::ProfileId,
        connector_reference_id: &str,
    ) -> StorageResult<Self> {
```

Or:
```suggestion
    pub async fn find_by_connector_reference_id_profile_id(
        conn: &PgPooledConn,
        connector_reference_id: &str,
        profile_id: &common_utils::id_type::ProfileId,
    ) -> StorageResult<Self> {
```

I'd prefer the former. Then fix the param order in the usages as well.